common/install_android.sh

```diff
@@ -2,13 +2,15 @@
 
 set -ex
 
+[ -n "${ANDROID_NDK}" ] || ANDROID_NDK=r13b
```

Comment:
you should just make this r19c

Reply:
ok, I'll just remove ANDROID_NDK=r13b - I was trying to keep in sync with caffe2's: https://github.com/pytorch/pytorch/blob/master/docker/caffe2/jenkins/common/install_android.sh
libcaffe2 still needs r13b to build and I don't want to break the caffe2 android CircleCI job there. We don't really need a default here since we don't have any pytorch android CI job yet.

Reply:
i can confirm that with https://github.com/pytorch/pytorch/pull/15509 I've seen libcaffe2 build fine with latest Android ndk